src/commands/BootstrapCommand.js

```diff
@@ -27,7 +40,7 @@ export default class BootstrapCommand extends Command {
     this.progressBar.init(this.packages.length);
     this.logger.info("Linking all dependencies");
 
-    async.parallelLimit(this.packages.map(pkg => done => {
+    async.parallelLimit(this.packages.filter(pkg => !pkg.ignored).map(pkg => done => {
```

Comment:
> I haven't really thought through the implementation. However, removing it from the packageGraph means you just never have access to that information which could cause problems down the road when we try to implement something else so I guess I would prefer that it didn't.

My comment here was about completely removing packages that have been ignored from the packageGraph. Which you have not done here.

I'm more concerned about having the logic for checking for ignored packages inside `PackageUtilities.getPackages` and `Package.ignored`. Those functions shouldn't have to be modified in order to implement this.